### DISCUSSION TITLE
Is it possible to treat lint warnings as errors?

### DISCUSSION THREAD

**Discussion by texuf** [vercel/turborepo#4360] [Help] [answered]
It looks like eslint has a `--max-warnings=0` flag that I can pass to treat warnings as errors. 
Does anything like that exist for turbo? I'm currently calling `turbo lint` but I don't see a matching flag or a way to pass on the --max-warnings flag

Thank you!

**Comment by mehulkar** [comment, marked as answer]
`turbo lint` has the effect of running the `lint` script in all your packages. You can look at each package.json in your repo to see what that `lint` script is doing. If it's using `eslint`, you can add the `--max-warnings` flag to the package.json definition.

**Reply by satyarth-shankar-foodsmart** [reply]
thank you for the elegant solution
